docs/modules/ROOT/partials/cmos-health-checks.adoc

```diff
@@ -195,4 +195,15 @@ Alternatively, navigate to your Data storage directory and enter the following c
 
 *Further Reading*: https://docs.couchbase.com/server/current/learn/data/data.html#metadata[Metadata], https://docs.couchbase.com/server/current/rest-api/rest-bucket-create.html#evictionpolicy[evictionPolicy].
 
+[#CB90061]
+=== Disk Write Queue (CB90061)
+
+*Background*: Having items in the disk write queue for too long could cause backoffs. This usually indicates either a sizing issue, or a storage medium failing.
+
+*Condition*: Age of items in disk write queue over 40 seconds.
+
+*Remediation*: Review your hardware for malfunctions or sizing issues. If the problem persists, then please contact Couchbase Technical Support. 
+
+*Further Reading*: https://docs.couchbase.com/server/current/metrics-reference/data-service-metrics.html#kv_ep_storage_age_seconds, https://docs.couchbase.com/server/current/install/sizing-general.html#sizing-data-service-nodes
```

Comment:
Obvs correct me if I'm wrong but does the hashtag do the same as the square brackets?

Reply:
in a URL a hash tag links to a specific part of the web page, in this case its the section on kv_ep_storage_age_seconds

Reply:
```suggestion
*Further Reading*: https://docs.couchbase.com/server/current/metrics-reference/data-service-metrics.html#kv_ep_storage_age_seconds[kv_ep_storage_age_seconds], https://docs.couchbase.com/server/current/install/sizing-general.html#sizing-data-service-nodes[sizing-data-service-nodes]
```

ahh okay, probs would be better to have something like this then to look nicer

Reply:
btw these URI links will work, but in our docs we use xref to link to other internal docs pages, following convention in https://docs.antora.org/antora/2.2/asciidoc/page-to-page-xref/

So https://docs.couchbase.com/server/current/metrics-reference/data-service-metrics.html#kv_ep_storage_age_seconds
would be
xref:7.0@server:metrics-reference:data-service-metrics.adoc#kv_ep_storage_age_second[link name in here].

Not critical to get these changed before the release, but could they be used in new docs?

Thanks

Reply:
@RichardSmedley Ah, thanks for the pointer, I'll file a CMOS to get that sorted out across all our docs.

@BabarKhanDev With that change it renders like this - should we make the link text be the actual title, instead of the hyphenated file name?
![image](https://user-images.githubusercontent.com/2904440/151189083-1d9bcf49-e6a4-4896-b16f-29666281ad38.png)

Reply:
@RichardSmedley Thank you, I'll make sure to follow that convention

Reply:
@markspolakovs Yeah I think we should make the text the actual title